experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/editableTree.events.spec.ts

```diff
@@ -654,3 +659,70 @@ describe("beforeChange/afterChange events", () => {
 		assert.strictEqual(childAfterCounter, 1);
 	});
 });
+
+describe("onNextChange event", () => {
+	const sb = new SchemaBuilder({ scope: "test" });
+	const object = sb.object("object", { content: sb.number });
+	const schema = sb.intoSchema(object);
+	const initialTree = { content: 3 };
+
+	it("fires exactly once after a change", () => {
+		const view = viewWithContent({ schema, initialTree });
+		const editNode = getEditNode(view.root);
+		let changeCount = 0;
```

Comment:
Nit: this variable name confused me for a moment. Would something like `onNextChangeCount` or just `callCount` be more clear?

Reply:
(100% nitpick - feel free to ignore)

Reply:
Nah, it's a good nitpick. Updated.